userspace/libsinsp/threadinfo.cpp

```diff
@@ -927,29 +977,6 @@ void sinsp_threadinfo::set_cwd(const char* cwd, uint32_t cwdlen)
 	}
 }
 
-void sinsp_threadinfo::allocate_private_state()
```

Comment:
Could we remove the `m_private_state` field in `sinsp_threadinfo` class and all its related methods?